djvm/src/main/kotlin/sandbox/java/lang/DJVM.kt

```diff
@@ -84,17 +84,31 @@ fun Throwable.escapeSandbox(): kotlin.Throwable {
                 initCause(it.escapeSandbox())
             }
             stackTrace = copyFromDJVM(this@escapeSandbox.stackTrace)
+            sanitise(0)
             this@escapeSandbox.suppressed.forEach {
                 addSuppressed(it.escapeSandbox())
             }
         }
     } catch (e: Exception) {
-        RuleViolationError(e.message).sanitise()
+        RuleViolationError(e.message).sanitise(1)
     }
 }
 
 private fun Array<*>.fromDJVMArray(): Array<*> = Object.fromDJVM(this)
 
+/**
+ * Throws a [RuleViolationError] to exit the sandbox.
+ * This function never returns, and we can inform the
+ * caller not to expect us to by invoking it via:
+ *
+ *     throw DJVM.fail("message")
+ */
+fun fail(message: kotlin.String): Error {
```

Comment:
Kotlin defines the type `Nothing` as the return type of non returning functions

Reply:
True, and I started with that. (Apparently it maps to `java.lang.Void` in the bytecode, as opposed to `V`.) However, `Nothing` is a Kotlin feature and I need `DJVM.fail(String)` to be invokable from pure Java.